### PR TITLE
Drug rehabilitation htmlform

### PR DESCRIPTION
**Pull request by Dutervil**
@lnball I have finished create the field in this form. please have a look

### CODE REVIEW

File: configuration/pih/htmlforms/drugRehab.xml [hunk 1]
```diff
@@ -214,11 +218,69 @@
         </includeIf>
     </ifMode>
 
-    <section id="consult" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="consult">
-        <div class="section-container">
-            <label>
-                <uimessage code="add form here" />
-            </label>
+    <section id="mh_scores" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.drugRehab">
```
Comment (lnball) [inline configuration/pih/htmlforms/drugRehab.xml:221]: Change the id.  "mh_scores" is old.  Otherwise it looks good @Dutervil

**Review by lnball — CHANGES_REQUESTED** [review]
One tiny suggestion

File: configuration/pih/htmlforms/drugRehab.xml (hunk 1 above)
Comment (Dutervil) [reply configuration/pih/htmlforms/drugRehab.xml:221]: I can change it to anything ? @lnball

Comment (Dutervil) [reply configuration/pih/htmlforms/drugRehab.xml:221]: it anything it fine, you can merge for testing @lnball

Comment (lnball) [reply configuration/pih/htmlforms/drugRehab.xml:221]: I normally change the id to what's in the section.  You call the section "Drug Rehab"?   Will there only be 1 section on the form?

Comment (Dutervil) [reply configuration/pih/htmlforms/drugRehab.xml:221]: One Section for now

**Review by lnball — APPROVED** [review]
I am merging in your PR.

**Pull request merged**